src/ownership/copy-clone.md

```diff
@@ -29,3 +29,20 @@ fn main() {
 
 * After the assignment, both `p1` and `p2` own their own data.
 * We can also use `p1.clone()` to explicitly copy the data.
+
+<details>
+
+Copying and cloning are not the same thing:
+
+* Copying refers to bitwise copies of memory regions and does not work on arbitrary objects.
+* Cloning is a more general operation and also allows for custom behavior by implementing the `Clone` trait.
+* Copying does not work on types that implement the `Drop` trait.
+
+In the above example, try the following:
+
+* Try printing `p1` twice after the move. What do you expect to happen?
```

Comment:
Why twice?

Reply:
Good catch, was changing things in the playground and didn't notice that the original example is already good enough. Removed that whole bullet point.